perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -475,6 +475,15 @@ sub add_report : Private {
     my $report = $c->stash->{report};
     $report->confirm;
     $report->update;
+
+    $c->model('DB::Alert')->find_or_create({
+        user => $report->user,
+        alert_type => 'new_updates',
+        parameter => $report->id,
+        cobrand => $report->cobrand,
+        lang => $report->lang,
+    })->confirm;
```

Comment:
Do we want to offer a way to not get alerts?

Reply:
We don't on .com for normal reports :) and I'd say you're more likely to want to know about these type of reports.